attributes/network/11_proof.adoc

```diff
@@ -0,0 +1,32 @@
+= Proof Attribute (#11)
+:cddl: ./cddl/
+
+This attribute allows server to deliver the proof of an object requested.
+The proof itself is a sequence of actions used to construct the state in
+which the object is embedded within the corresponding Merkle tree.
+
+== Params
+
+== Returns
+
+A proof is a collection of operations used to reconstruct a state that can
+be embedded into the existing state of the Merkle tree that contains it.
+
+Each state can be represented by one of five operations:
+
+* A node hash
+
+* A key/value hash of a node
+
+* A key and value pair of a node
+
+* The "parent" operation
+
+* The "child" operation
+
+Interpreting these operations as acting on a stack, the first three operations listed
+can be interpreted as pushing data onto the stack.  In the parent operation,
+two items are popped from the stack, and a new item with the second item as the
+left child of the first item is pushed onto the stack.  In the child operation,
+two items are popped from the stack, and a new item with the first item as the right child
+of the second item is pushed back onto the stack.
```

Comment:
I like the way they present it in https://github.com/liftedinit/merk/blob/develop/docs/algorithms.md#binary-format

Their example makes everything very clear.